bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/MiIoQuantiyTypes.java

```diff
@@ -33,31 +36,49 @@
 @NonNullByDefault
 public enum MiIoQuantiyTypes {
 
-    CELCIUS(SIUnits.CELSIUS),
-    FAHRENHEIT(ImperialUnits.FAHRENHEIT),
-    SECOND(SmartHomeUnits.SECOND),
-    MINUTE(SmartHomeUnits.MINUTE),
-    HOUR(SmartHomeUnits.HOUR),
-    SECONDS(SmartHomeUnits.SECOND),
-    MINUTES(SmartHomeUnits.MINUTE),
-    HOURS(SmartHomeUnits.HOUR),
-    AMPERE(SmartHomeUnits.AMPERE),
-    WATT(SmartHomeUnits.WATT);
+    CELCIUS(SIUnits.CELSIUS, "C"),
+    FAHRENHEIT(ImperialUnits.FAHRENHEIT, ""),
+    SECOND(SmartHomeUnits.SECOND, "seconds"),
+    MINUTE(SmartHomeUnits.MINUTE, "minutes"),
+    HOUR(SmartHomeUnits.HOUR, "hours"),
+    AMPERE(SmartHomeUnits.AMPERE, ""),
+    WATT(SmartHomeUnits.WATT, ""),
+    SQUARE_METRE(Units.SQUARE_METRE, "square_meter,squaremeter"),
+    PERCENT(SmartHomeUnits.PERCENT, "");
 
     private final Unit<?> unit;
+    private final String aliasses;
 
     private static Map<String, Unit<?>> stringMap = Arrays.stream(values())
             .collect(Collectors.toMap(Enum::toString, MiIoQuantiyTypes::getUnit));
 
-    private MiIoQuantiyTypes(Unit<?> unit) {
+    private static Map<String, Unit<?>> aliasMap() {
+        Map<String, Unit<?>> aliassesMap = new HashMap<>();
+        for (MiIoQuantiyTypes miIoQuantiyType : values()) {
+            for (String alias : miIoQuantiyType.getAliasses().split(",")) {
+                aliassesMap.put(alias.toLowerCase(), miIoQuantiyType.getUnit());
+            }
+        }
+        return aliassesMap;
+    }
+
+    private MiIoQuantiyTypes(Unit<?> unit, String aliasses) {
```

Comment:
I see no reason that you need to have your aliases comma separated when you can just have them as an array instead.
```suggestion
    private MiIoQuantiyTypes(Unit<?> unit, String... aliasses) {
```